node.gyp

```diff
@@ -661,13 +661,18 @@
         # When node is built as a shared library on windows the
         # dll will export a number of symbols which are also
         # available in openssl.lib leading to link time errors.
-        # For the C++ unit tests it is not necessary to link against
-        # the dll. When msbuild processes node.sln (solution file) it
-        # will build the node target before running this target so we
-        # can safely exclude the dependency here.
+        # In this particular case we will only link against the
+        # dll.
         ['OS=="win" and node_shared=="true"', {
-          'dependencies!': [
+          'dependencies=': [
             '<(node_core_target_name)',
+            'deps/gtest/gtest.gyp:gtest',
+          ],
+          'dependencies!': [
```

Comment:
Just checked, these 5 lines are unnecessary to this approach (only `dll` & `gtest`).